Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp

```diff
@@ -310,9 +310,6 @@ void WebsiteDataStore::resolveDirectoriesIfNecessary()
         m_resolvedConfiguration->setCookieStorageFile(resolveAndCreateReadWriteDirectoryForSandboxExtension(FileSystem::parentPath(m_configuration->cookieStorageFile())));
         m_resolvedConfiguration->setCookieStorageFile(FileSystem::pathByAppendingComponent(m_resolvedConfiguration->cookieStorageFile(), FileSystem::pathFileName(m_configuration->cookieStorageFile())));
     }
-
-    if (auto directory = m_configuration->webSQLDatabaseDirectory(); !directory.isEmpty())
-        FileSystem::deleteNonEmptyDirectory(directory);
```

Comment:
Should we have a bug tracking removal of webSQLDatabaseDirectory and related code?

Reply:
Sure, filed https://bugs.webkit.org/show_bug.cgi?id=241684.